public/static/docs/command-reference/diff.md

```diff
@@ -46,148 +57,142 @@ for example when `dvc init` was used with the `--no-scm` option.
 
 ## Examples
 
-For these examples we can use the chapters in our
-[Get Started](/doc/get-started) section, up to
-[Add Files](/doc/get-started/add-files).
+For these examples we can use the [Get Started](/doc/get-started) repository.
 
 <details>
 
-### Click and expand to setup example
+### Click and expand to setup the examples project
 
-Start by cloning our example repo if you don't already have it. Then move into
-the repo and checkout the
-[3-add-file](https://github.com/iterative/example-get-started/releases/tag/3-add-file)
-tag, corresponding to the [Add Files](/doc/get-started/add-files) _Get Started_
-chapter:
+Start by cloning our example repo if you don't already have it:
 
 ```dvc
 $ git clone https://github.com/iterative/example-get-started
 $ cd example-get-started
-$ git checkout 3-add-file
 ```
 
-Download the precomputed data using:
+Download data using:
 
 ```dvc
-$ dvc pull
+$ dvc fetch -T
 Preparing to download data from 'https://remote.dvc.org/get-started'
 ...
 ```
 
+The `-T` flag passed to `dvc fetch` makes sure we have all the data files
+related to all existing tags in the repo. You take a look at the
+[available tags](https://github.com/iterative/example-get-started/tags) of our
+example repo.
+
 </details>
 
-## Example: Previous commit in the same branch
+## Example: Checking workspace changes
 
 The minimal `dvc diff`, run without arguments, defaults to comparing DVC-tacked
 files between `HEAD` (current Git commit) and the current <abbr>workspace</abbr>
-(uncommitted changes, if any).
-
-To see the difference between the very previous commit of the project and the
-workspace, we can use `HEAD^` as `a_ref`:
+(uncommitted changes, if any):
 
 ```dvc
-$ dvc diff HEAD^
-dvc diff from df613bc to ed10968
-
-diff for 'data/data.xml'
-+data/data.xml with md5 a304afb96060aad90176268345e10355
-
-added file with size 37.9 MB
+$ dvc diff
 ```
 
-## Example: Specific targets across Git commits
+> It produces an empty result if there are no changes to any files or
+> directories in the current workspace.
```

Comment:
No need for block quote, I think.

Reply:
And maybe
`... in the current workspace, which we just cloned and pulled.`
?

Reply:
going to remove this, tbh ... don't like there back-references to the setup instructions ... ideally examples should be self descriptive. Will do another update when I deal with Katacoda.